app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SponsorsFragment.java

```diff
@@ -81,7 +78,11 @@ private void initView() {
         binding.recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
     }
 
-    private static class SponsorAdapter
```

Comment:
mmm... probably, I still need to study more.
static class can not access non-static method. It have to use non-static method `getActivity()` to open CustomTabs.
What is best practice to fix these cases?